examples/FsCheck.NUnit.Examples/FsCheck.NUnit.Examples.fsproj

```diff
@@ -1,18 +1,22 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
-  <PropertyGroup>
-    <AssemblyName>FsCheck.NUnit.Examples</AssemblyName>
-    <TargetFramework>net6.0</TargetFramework>
-    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
-  </PropertyGroup>
-  <ItemGroup>
-    <Content Include="App.config" />
-    <Compile Include="PropertyExamples.fs" />
-    <None Include="paket.references" />
-  </ItemGroup>
-  <ItemGroup>
-    <ProjectReference Include="../../src/FsCheck/FsCheck.fsproj" />
-    <ProjectReference Include="../../src/FsCheck.NUnit/FsCheck.NUnit.fsproj" />
-  </ItemGroup>
-  <Import Project="..\..\.paket\Paket.Restore.targets" />
+    <PropertyGroup>
+        <AssemblyName>FsCheck.NUnit.Examples</AssemblyName>
+        <TargetFramework>net6.0</TargetFramework>
+        <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
+    </PropertyGroup>
+    <ItemGroup>
+        <Content Include="App.config"/>
+        <Compile Include="PropertyExamples.fs"/>
+    </ItemGroup>
+    <ItemGroup>
+        <PackageReference Include="Microsoft.NET.Test.Sdk"/>
```

Comment:
See examples/FsCheck.NUnit.Examples/paket.references; I've accidentally deleted the FSharp.Core reference from here but it's picked up through FsCheck anyway